pages/school-administrators.md

```diff
@@ -389,6 +390,62 @@ Use these resources to get training and boost your skills to help support milita
 </section>
 <div>
 
+
+<h2 id="program-approvals" tabindex="-1">Program Approvals</h2>
+<ul class="va-nav-linkslist-list">
+  <li>
+    <span><va-link
+      active
+      href="https://benefits.va.gov/gibill/School_Program_Approval.asp"
+      text="Domestic School Program Approvals"
+    /></span>
+    <p class="va-nav-linkslist-description">Identify general approval guidelines for programs of education broken down by various types of training programs.</p>
+  </li>
+  <li>
+    <span><va-link
+      active
+      href="https://www.benefits.va.gov/gibill/foreign_program_approval_information_for_schools.asp"
+      text="Foreign School Approval Process"
+    /></span>
+    <p class="va-nav-linkslist-description">Identify general approval guidelines for programs of education offered at foreign institutions.</p>
+  </li>
+  <li>
+    <span><va-link
+      active
+      href="https://benefits.va.gov/gibill/federalemployerOJTandApprenticeshipProgramApprovalInformation.asp"
+      text="Federal On The Job Training/ Apprenticeship Approval"
+    /></span>
+    <p class="va-nav-linkslist-description">Discover the process and criteria for VA approval of on-the-job training and apprenticeship programs</p>
```

Comment:
Notice all the other paragraphs have a period at the end. Do we want one on this one too? Definitely not a reason to keep this from moving forward.